### PR TITLE
locari.jp

### PR DESCRIPTION
**Pull request by jellizaveta**
https://github.com/AdguardTeam/AdguardFilters/issues/118399

<details><summary>Screenshot:</summary>

Снимок экрана 2022-05-12 в 14 32 33

Снимок экрана 2022-05-12 в 14 32 49

Снимок экрана 2022-05-12 в 14 33 08

</details><br/>

### CODE REVIEW

File: JapaneseFilter/sections/specific.txt [hunk 1]
```diff
@@ -42,6 +42,8 @@ aviationwire.jp,getnavi.jp,entamerush.jp,gourmetpress.net,career-find.jp,mainich
 bridge-salon.jp##div[id^="jp_ir_"][style]
 blkt.net##.ad001
 blkt.net##.mkjAd
+locari.jp##.footer__banner
```
Comment (Yuki2718) [inline JapaneseFilter/sections/specific.txt:45]: This is PR for advertiser who may want their ads to be put on the site. Should we block this on main list? @Alex-302 @zloyden

File: AnnoyancesFilter/sections/mobile-app.txt [hunk 2]
```diff
@@ -25,6 +25,7 @@
 m.guancha.cn##.swiper-slide
 m.guancha.cn##.g_swiper_container
 m.guancha.cn##.g_sea
+locari.jp##.app-download-banner
```
Comment (Yuki2718) [inline AnnoyancesFilter/sections/mobile-app.txt:28]: `locari.jp##.in-app` can be added (IMO optional though).

File: JapaneseFilter/sections/specific.txt (hunk 1 above)
Comment (zloyden) [reply JapaneseFilter/sections/specific.txt:45]: I don't think that we should remove it from the footer.

**Review by zloyden — APPROVED** [review]

**Comment by Yuki2718** [comment]
Oh, no need to remove `locari.jp##.app-download-banner`. `locari.jp##.in-app` is just another app banner in middle of the article.

**Pull request merged**